libs/mutantschemata/cpp_source/rewrite.hpp

```diff
@@ -0,0 +1,794 @@
+//------------------------------------------------------------------------------
+// Mutant schemata using Clang rewriter and RecursiveASTVisitor
+//
+// Sten Vercammem (sten.vercammen@uantwerpen.be)
+//------------------------------------------------------------------------------
+
+#include <set>
+#include <sstream>
+#include <vector>
+#include <fstream>
+#include <functional>
+
+#include "clang/AST/AST.h"
+#include "clang/AST/ASTContext.h"
+#include "clang/AST/ASTConsumer.h"
+#include "clang/AST/RecursiveASTVisitor.h"
+#include "clang/Frontend/ASTConsumers.h"
+#include "clang/Frontend/FrontendActions.h"
+#include "clang/Frontend/CompilerInstance.h"
+#include "clang/Tooling/CommonOptionsParser.h"
+#include "clang/Tooling/Tooling.h"
+#include "clang/Rewrite/Core/Rewriter.h"
+
+#ifndef MS_CLANG_REWRITE
+#define MS_CLANG_REWRITE
+
+
+// defines to control what happens \\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\|
+// config mutant schemata
```

Comment:
It seems that these booleans are used as flags. A more maintainable approach would be to assign them via macros passed via the -D flag when building dextool, then they could also be tested in separation